### PR TITLE
Feature/structure

### PR DESCRIPTION
**Pull request by HFG43**
Hi, thanks for your previous feedback. As requested, I have added the License link to the Readme file

Hi there! 🙌🙌

 _Please consider the general issue with Webhint Linter_ 

I'm sending this PR for you to review my code. 🐱‍👤🐱‍👤 Thank you in advance for your feedback!
For this feature, I have added the following:

- [ ] Use Linters ✔
- [ ] Use Webpack ✔
- [ ] Follow Git Flow ✔
- [ ] Set the structure for the LeaderBoard project ✔
- [ ] Follow the wireframe to set the structure ✔
- [ ] Add an array with objects to fill un UL with each list item containing name and score ✔
- [ ] Add README file ✔

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @HFG43 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Kindly note that it is important that all your readme headers are not empty for professionalism. Kindly add a link to your `MIT` License

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,174 @@
...
+git clone https://github.com/HFG43/Leaderboard
+
+### Install
+
+Open index.html file with your Code Editor, and review the behaviour of the page under different sizes.
+
+### Usage
+
+To run the project, execute the following command:
+
```
Comment (topeogunleye) [inline README.md:108]: - [ ] I kindly suggest that you add the step for usage of your web app. for example
[OPTIONAL]
```
### Usage

To run the project, execute the following command:


- Start the development server
`   ```sh
`   npm start
`   ```
`
```

**Review by topeogunleye — APPROVED** [review]
Hi @hagan199, 

Congratulations on completing this project! You have done a great job of setting up a JavaScript project for the Leaderboard list app, using webpack and ES6 features. You have followed all the project requirements and created a basic project structure. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

dwayne-the-rock

## (Highlights) Good Points: 💪
- You have used webpack to bundle your JavaScript files and included all the necessary packages in your package.json file as devDependencies. Well done!
- You have added npm scripts to run, build, and start your project. Good job!
- You have added node_modules folder to your .gitignore file. Nice work!
- You have created a dist folder that contains your bundled JavaScript file and your index.html file. Excellent!
- You have created a src folder that contains your style.css and index.js files. Awesome!
- You have written HTML code that matches the wireframe provided. Superb!

You should be proud of your work and the skills you have learned in this project. Keep up the good work and happy coding!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
_Please kindly check my review comments_

- [ ]  Next time, and for future commits, I kindly suggest that you complete your commit messages.
:x: Bad Commit Messages :x:

image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HFG43** [comment]
@topeogunleye Many thanks for your feedback! I will apply all your suggestions in the incoming tasks

**Pull request merged**